pandas/tests/frame/constructors/test_from_dict.py

```diff
@@ -165,7 +166,7 @@ def test_constructor_from_dict_tuples(self, data_dict, keys, orient):
         df = DataFrame.from_dict(data_dict, orient)
 
         result = df.columns
-        expected = Index(keys, dtype="object", tupleize_cols=False)
+        expected = Index(keys, tupleize_cols=False) if keys else RangeIndex(0)
```

Comment:
any chance we could include this in the parametrisation, instead of introducing an if-else in the test?

Reply:
Yes, that would be better.